test/e2e/main_test.go

```diff
@@ -47,6 +47,7 @@ var (
 	postUpgradeTestDirName          = "post-upgrade"
 	gatherTestArtifactsScript       = "gather-test-install-artifacts.sh"
 	testMeteringConfigManifestsPath = "/test/e2e/manifests/meteringconfigs/"
+	testNFSManifestPath 			= "/test/e2e/manifest/nfs/"
```

Comment:
This is the wrong path. It's `test/e2e/manifests/nfs` and not `test/e2e/*manifest*/nfs`